server/app.js

```diff
@@ -1,5 +1,6 @@
 const express = require('express');
 const pg = require('../database1/postgresModel');
+const cass = require('../database2/cassandraModel')

```

Comment:
Any thoughts on how you'll ultimately want to switch between databases? Commenting out lines is one way to do it. Currently I'm working toward just having two different server app files, and just commenting out the unused one on my server index.js, so there's only one thing to change at a time, but ultimately this is probably best done with a .env file. My comment here is that requiring both Postgres and Cassandra will actually connect to both at the same time, since importing a file runs it. You may want to just comment out whichever you aren't using at the time.

Reply:
I def need to set up a dotenv file but I don't know if I will use it to convert between since commenting out is pretty easy and we're going to eliminate one of the two pretty quickly here. I have the controller I'm not using commented out in each request router since they are each just one line, but I like the idea of commenting out the require at the top to keep things clear.

Reply:
Thanks for all the thoughtful feedback Stu! Super helpful!